Add F86XK_Forward and XF86XK_Back for page switch by muhpdf@binkmail.com 
on http://bugs.ghostscript.com/show_bug.cgi?id=693494

--- mupdf-1.3/platform/x11/x11_main.c
+++ mupdf-1.3/platform/x11/x11_main.c
@@ -5,6 +5,7 @@
 #include <X11/Xatom.h>
 #include <X11/cursorfont.h>
 #include <X11/keysym.h>
+#include <X11/XF86keysym.h>
 
 #include <sys/select.h>
 #include <sys/time.h>
@@ -836,9 +837,11 @@
 						break;
 
 					case XK_Page_Up:
+					case XF86XK_Back:
 						len = 1; buf[0] = ',';
 						break;
 					case XK_Page_Down:
+					case XF86XK_Forward:
 						len = 1; buf[0] = '.';
 						break;
 					}
